### PR TITLE
Fix: add parameter null check

### PR DESCRIPTION
**Pull request by Jim8y**
# Description

This pr follows https://github.com/neo-project/neo/pull/3089 to thoroughly check the neo core parameter check.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Neo/SmartContract/ApplicationEngine.Runtime.cs [hunk 1]
```diff
@@ -209,6 +209,7 @@ protected internal StackItem GetScriptContainer()
         /// </summary>
         protected internal void RuntimeLoadScript(byte[] script, CallFlags callFlags, Array args)
```
Comment (cschuchardt88) [inline src/Neo/SmartContract/ApplicationEngine.Runtime.cs:210]: Dont forget `args`. That can be `null` too.

```
PUSHNULL
PUSHNULL
PUSHNULL 
SYSCALL "System.Runtime.LoadScript"
```

File: src/Neo/SmartContract/ApplicationEngine.Storage.cs [hunk 2]
```diff
@@ -126,6 +126,7 @@ protected internal static StorageContext AsReadOnly(StorageContext context)
         /// <returns>The value of the entry. Or <see langword="null"/> if the entry doesn't exist.</returns>
         protected internal ReadOnlyMemory<byte>? Get(StorageContext context, byte[] key)
```
Comment (cschuchardt88) [inline src/Neo/SmartContract/ApplicationEngine.Storage.cs:127]: `StorageContext` `null` check?

**Review by shargon — COMMENTED** [review]
We need neo project as nullable

**Review by shargon — COMMENTED** [review]
We can do it better with nullable checks

**Pull request closed without merging**